shared/qcommon/q_math.h

```diff
@@ -180,6 +184,29 @@ typedef struct cplane_s {
 	byte	type;			// for fast side tests: 0,1,2 = axial, 3 = nonaxial
 	byte	signbits;		// signx + (signy<<1) + (signz<<2), used as lookup during collision
 	byte	pad[2];
+
+
+#ifdef __cplusplus
```

Comment:
Saving/loading game files shouldn't be in this file. This file is shared by the MP game as well now, and I'd like the shared/ folder to be agnostic of SP or MP.